lib/openapi2ruby/openapi/schema/property.rb

```diff
@@ -8,6 +8,14 @@ def initialize(content)
       @items = content[:definition]['items']
       @format = content[:definition]['format']
       @ref = content[:definition]['$ref']
+      if content[:definition]['oneOf']
+        @type = 'oneOf'
+        @one_of_refs = content[:definition]['oneOf'].map do |content|
+          content['$ref'].split('/').last
+        end
+      else
+        @one_of_items = []
```

Comment:
This ivar is not used everywhere.
I prefer to use same name above.
```suggestion
        @one_of_refs = []
```